gem/lib/earlgrey/configure_earlgrey.rb

```diff
@@ -364,8 +379,7 @@ def has_swift?(target)
     # @param [Xcodeproj::Project] project
     # @param [PBXNativeTarget] target
     def copy_swift_files(project, target, swift_version = nil)
-      return unless has_swift?(target) || !swift_version.nil?
-
+      return unless has_swift?(target) || !swift_version.to_s.empty?
```

Comment:
I don't know the semantic meaning of Unspecified or empty.

my guess is if you have swift files but doesn't specify its version, it would throw an error when compiling?